targets/frontend/src/components/storage/DropZone.js

```diff
@@ -0,0 +1,58 @@
+/** @jsx jsx */
+
+import PropTypes from "prop-types";
+import { useCallback, useState } from "react";
+import { useDropzone } from "react-dropzone";
+import { mutate } from "swr";
+import { jsx, Spinner } from "theme-ui";
+
+const defaultStyles = {
+  border: "2px dotted silver",
+  borderRadius: "small",
+  p: "medium",
+};
+
+export function DropZone({ onDrop: onDropCallback, customStyles }) {
+  const [uploading, setUploading] = useState(false);
+  const onDrop = useCallback(
+    async (acceptedFiles) => {
+      setUploading(true);
+      const formData = new FormData();
+      for (const i in acceptedFiles) {
+        if (acceptedFiles[i] instanceof File) {
+          console.log(acceptedFiles[i].path);
+          console.log(acceptedFiles[i]);
+          formData.append(acceptedFiles[i].path, acceptedFiles[i]);
+        }
+      }
+      onDropCallback(formData).finally(() => {
+        setUploading(false);
+        mutate("files");
+      });
+    },
+    [onDropCallback]
+  );
+
+  const { getRootProps, getInputProps, isDragAccept } = useDropzone({
+    accept:
+      "image/jpeg, image/png, application/pdf, application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+    onDrop,
+  });
+
+  if (isDragAccept) {
+    defaultStyles.background = "#d1ffd9"; // some light green
```

Comment:
I think we can put the color in theme.

Reply:
gonna change for the existing positive green which is fine imho